ios/Stockfish/PGN.mm

```diff
@@ -30,17 +30,13 @@
 static const int TAB_SIZE = 8;
 static const int CHAR_EOF = 256;
 
-// types
-
-enum token_t {
```

Comment:
This enum syntax causes error "Definition of type 'token_t' conflicts with typedef of the same name".

I tried to change it to `typedef enum {...} token_t` but there's still error.